recipes/sqlite-zstd/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "sqlite-zstd" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/phiresky/sqlite-zstd/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: c29d5229dc2d6b7fad677c3115155918d0cb34897554675213f10cac8f1cb266
+
+build:
+  number: 0
+  skip: true  # [not linux] # macos will need >=10.12
```

Comment:
Do you want to add macOS? https://conda-forge.org/docs/maintainer/knowledge_base/#requiring-newer-macos-sdks

Reply:
Oh, I didn't know that bumping the SDK version was possible during staging. Testing in [e3a52e4](https://github.com/conda-forge/staged-recipes/pull/25862/commits/e3a52e4193170f1a8abe9e846b4a97c7288b37fb)